Fails to parse bookings

create one booking and try to list bookings

```
panic: parsing failure

goroutine 1 [running]:
sidus.io/boogrocha/internal/cli/commands.runList(0xc00015a500, 0xd81f48, 0x0, 0x0, 0x960f60)
        /home/gurgy/.cache/yay/boogrocha/src/boogrocha-r43.6342c69/internal/cli/commands/list.go:30 +0x6d7
sidus.io/boogrocha/internal/cli/commands.ListCmd.func1(0xc00015a500, 0xd81f48, 0x0, 0x0)
        /home/gurgy/.cache/yay/boogrocha/src/boogrocha-r43.6342c69/internal/cli/commands/list.go:18 +0x52
github.com/spf13/cobra.(*Command).execute(0xc00015a500, 0xd81f48, 0x0, 0x0, 0xc00015a500, 0xd81f48)
        /home/gurgy/go/pkg/mod/github.com/spf13/cobra@v0.0.4-0.20190321000552-67fc4837d267/command.go:766 +0x2ae
github.com/spf13/cobra.(*Command).ExecuteC(0xd5c760, 0x847ea3, 0xc000131f88, 0xc000096058)
        /home/gurgy/go/pkg/mod/github.com/spf13/cobra@v0.0.4-0.20190321000552-67fc4837d267/command.go:850 +0x2fc
github.com/spf13/cobra.(*Command).Execute(...)
        /home/gurgy/go/pkg/mod/github.com/spf13/cobra@v0.0.4-0.20190321000552-67fc4837d267/command.go:800
main.main()
        /home/gurgy/.cache/yay/boogrocha/src/boogrocha-r43.6342c69/cmd/bgc/main.go:9 +0x2e
```

--- a/internal/booking/chalmers/bookingService.go
+++ b/internal/booking/chalmers/bookingService.go
@@ -99,7 +99,7 @@ func (bs BookingService) MyBookings() ([]booking.Booking, error) {
 	bookings := make([]booking.Booking, 0, 4)
 	selectedDate := ""
 	for _, tr := range trs {
-		headline := tr.Find(".headline.changeDateLink")
+		headline := tr.Find(".headline.leftRounded.t")
 		if headline.Length() > 0 {
 			selectedDate = strings.Split(headline.Text(), " ")[1]
 		} else {

